native/cocos/platform/interfaces/modules/Device.h

```diff
@@ -72,6 +72,9 @@ class CC_DLL Device {
         float rotationRateGamma = 0.0F;
     };
 
+    static int getInnerWidth();
+    static int getInnerHeight();
+    static uint32_t getWindowHandler();
```

Comment:
Is it the pointer of window? If so, then 32 bit is not big enough on 64-bit systems.

Reply:
This only considers 32-bit, and it will be changed separately later

Reply:
https://github.com/cocos/3d-tasks/issues/13618